### PR TITLE
Dashboard/job insights

### PR DESCRIPTION
**Pull request by Gerhut**
@Anbang-Hu sorry to revert your markdown efforts, I found it should be listed and deal message by message.

If necessary I will split the revert commit into a separated PR.

image

- Currently the job insights "GPU idle" and "need more metrics" seems have different message levels (warn and info). They may have different styles for user if we can do that.
- It's better to give an action to user for each message, like "kill"(for gpu idle), "support"(maybe by default) or "dismiss"(cannot be persistent currently)

Expected fields:
- level
- message
- action

- [ ] drop the revert commit and rebase on the latest dev branch

### CODE REVIEW

File: src/docker-images/job-insighter/insight.py [hunk 1]
```diff
@@ -506,9 +508,9 @@ def upload_insights(insights, restful_url, dry_run):
             logger.exception("failed to upload insight for %s", job_id)
 
 
-def run(prometheus_url, mins_ago, restful_url, dry_run):
+def run(prometheus_url, hours_ago, restful_url, dry_run):
```
Comment (Anbang-Hu) [inline src/docker-images/job-insighter/insight.py]: Can you keep the change regarding `minutes`?

**Comment by Gerhut** [comment]
image

Deployed to int, seems the content is too much

**Comment by Anbang-Hu** [comment]
> image
> 
> Deployed to int, seems the content is too much

Let's use a collapseable section showing the first line with ellipses by default.

In addition, job insight should only show for running jobs.

**Comment by Gerhut** [comment]
image

**Comment by Gerhut** [comment]
image

---

image

**Comment by Gerhut** [comment]
changed to `SHOW ALL` to keep user from reminding of collapse from expand

**Pull request merged**